homeassistant/components/tankerkoenig/entity.py

```diff
@@ -12,14 +14,14 @@ class TankerkoenigCoordinatorEntity(CoordinatorEntity):
     _attr_has_entity_name = True
 
```

Comment:
```suggestion
    coordinator: TankerkoenigDataUpdateCoordinator
```
i know, this is existing code, but this would improve the typing

Reply:
Might as well just add the type to `CoordinatorEntity`